lib/elixir/lib/file.ex

```diff
@@ -946,8 +946,8 @@ defmodule File do
     * `:exclusive` - the file, when opened for writing, is created if it does
       not exist. If the file exists, open will return `{:error, :eexist}`.
 
-    * `:char_list` - when this term is given, read operations on the file will
-      return char lists rather than binaries.
+    * `:charlist` - when this term is given, read operations on the file will
```

Comment:
Just for another PR: :charlist is not an option in `mode/0` type, so we need to correct that.